### DISCUSSION TITLE
Change id of resultItem to one from the data source

### DISCUSSION THREAD

**Discussion by varna9000** [TarekRaafat/autoComplete.js#285] [Q&A]
I want to assign an `id` of `resultItem` with a value which comes from the Ajax data call. How do I do that?

**Comment by varna9000** [comment]
ok, I got it

```
resultItem: {
        	element: (item, data) => {
                    item.setAttribute("id", data.value.id);
     }
}
```

**Reply by mrelliot69** [reply]
how did you make it work with ajax?
im currently using the async but semms to slow
